omrmakefiles/omr.rc.in

```diff
@@ -0,0 +1,32 @@
+
+#include <windows.h>
+#include <winver.h>
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION 2,9,5,19691
+ PRODUCTVERSION 2,9,5,19691
+ FILEFLAGSMASK 0x3fL
+ FILEFLAGS 0x0L
+ FILEOS VOS_NT_WINDOWS32
+ FILETYPE VFT_DLL
+ FILESUBTYPE 0x0L
+BEGIN
+	BLOCK "StringFileInfo"
+	BEGIN
+		BLOCK "040904b0"
+		BEGIN
+			VALUE "CompanyName", @OMR_COPYRIGHT_WIN_COMPANYNAME@
+			VALUE "FileDescription", @OMR_COPYRIGHT_WIN_FILEDESCRIPTION@
+			VALUE "FileVersion", @OMR_COPYRIGHT_WIN_VERSION@
+			VALUE "InternalName", @OMR_COPYRIGHT_WIN_FILENAME@
+			VALUE "LegalCopyright", @OMR_COPYRIGHT_WIN_LEGALCOPYRIGHT@
+			VALUE "OriginalFilename", @OMR_COPYRIGHT_WIN_FILENAME@
+			VALUE "ProductName", @OMR_COPYRIGHT_WIN_PRODUCTNAME@
+			VALUE "ProductVersion", @OMR_COPYRIGHT_WIN_VERSION@
+		END
```

Comment:
* The substitution syntax looks correct 👍
* `CompanyName`, `LegalCopyright` should be statically set.
* Don't set fields if they're always blank--`InternalName`, `OriginalFilename`

Also note: you're still missing the `AC_ARG_VAR` macro calls in `configure.ac`. The configure variables won't be substituted without them.